aion/web/router.ex

```diff
@@ -42,6 +42,7 @@ defmodule Aion.Router do
     pipe_through [:api, :api_auth]
 
     get "/me", UserController, :get_user_info
+    get "/counts", RoomController, :get_counts
```

Comment:
I think it should be `/rooms/counts`.

Reply:
This one conflicts with what you suggested in the review. I'll try to go with a simple param first.